test/powerglitch.test.ts

```diff
@@ -43,7 +43,8 @@ const testAllElementTypes = (name: string, job: (elementType: string) => Promise
  * @param playing Whether all layers should be playing or not
  */
 const ensureAllLayersPlaying = (container: HTMLDivElement, playing: boolean) => {
-    for (const layerDiv of container.children) {
+    const layersContainer = container.firstElementChild as HTMLDivElement;
```

Comment:
Now, `containers` returned by `PowerGlitch.glitch` is now the entire container, which means that to get to the layers, you have to dig down one more level.

I won't consider this a breaking change as this `containers` was referenced nowhere until now in the usage guide and home page.